libpoolprotocols/getwork/EthGetworkClient.cpp

```diff
@@ -8,28 +8,31 @@ using namespace std;
 using namespace dev;
 using namespace eth;
 
-EthGetworkClient::EthGetworkClient(unsigned farmRecheckPeriod, bool submitHashrate) : PoolClient(), Worker("getwork"), m_submit_hashrate(submitHashrate)
+EthGetworkClient::EthGetworkClient(unsigned farmRecheckPeriod, bool submitHashrate)
+  : PoolClient(), Worker("getwork"), m_submit_hashrate(submitHashrate)
 {
-	m_farmRecheckPeriod = farmRecheckPeriod;
-	m_subscribed.store(true, std::memory_order_relaxed);
-	m_authorized.store(true, std::memory_order_relaxed);
-	if (m_submit_hashrate)
-		m_client_id = h256::random();
+    m_farmRecheckPeriod = farmRecheckPeriod;
+    m_subscribed.store(true, std::memory_order_relaxed);
+    m_authorized.store(true, std::memory_order_relaxed);
+    if (m_submit_hashrate)
+        m_client_id = h256::random();
 }
 
 EthGetworkClient::~EthGetworkClient()
 {
-	p_client = nullptr;
+    p_client = nullptr;
 }
 
 void EthGetworkClient::connect()
 {
-    std::string uri = "http://" + m_conn->Host() + ":" + to_string(m_conn->Port());
+    stringstream ss;
```

Comment:
Why back to stringstream?

Reply:
Ouch ... it's likely I was not in sync.
Correcting